core-services/image-mirroring/openshift/mapping_origin_4_9

```diff
@@ -98,6 +98,7 @@ registry.ci.openshift.org/origin/4.9:descheduler quay.io/openshift/origin-desche
 registry.ci.openshift.org/origin/4.9:docker-builder quay.io/openshift/origin-docker-builder:4.9 quay.io/openshift/origin-docker-builder:4.9.0
 registry.ci.openshift.org/origin/4.9:docker-registry quay.io/openshift/origin-docker-registry:4.9 quay.io/openshift/origin-docker-registry:4.9.0
 registry.ci.openshift.org/origin/4.9:driver-toolkit quay.io/openshift/origin-driver-toolkit:4.9 quay.io/openshift/origin-driver-toolkit:4.9.0
+registry.ci.openshift.org/origin/4.9:driver-toolkit-test quay.io/openshift/origin-driver-toolkit-test:4.9 quay.io/openshift/origin-driver-toolkit-test:4.9.0
```

Comment:
driver-toolkit-tests

(we use plural “tests”)

Reply:
Done, thanks.